### DISCUSSION TITLE
PO number on invoice

### DISCUSSION THREAD

**Discussion by extect** [YetiForceCompany/YetiForce#47] [How to]
When a customer is ordering from me, they require me to put their PO number on the invoice I send them. Since this is quite a common procedure, I am sure there is a way to store my customer's PO number in Yetiforce and output it on my invoice. Any idea how I can do this?

**Comment by Ilamparithi-Tamizhanangu** [comment]
Create the custom text field for the PO reference number and use the same in the PDF output.
